src/main/webapp/app/admin/system-notification-management/system-notification-management.route.ts

```diff
@@ -16,7 +18,10 @@ export class SystemNotificationManagementResolve implements Resolve<any> {
      */
     resolve(route: ActivatedRouteSnapshot) {
         if (route.params['id']) {
-            return this.service.find(parseInt(route.params['id'], 10));
+            return this.service.find(parseInt(route.params['id'], 10)).pipe(
+                filter((response: HttpResponse<SystemNotification>) => response.ok),
+                map((exam: HttpResponse<SystemNotification>) => exam.body!),
```

Comment:
Why is this called exam? It seems to be an HttpRespone with a notification, no exams involved

Reply:
just a copy and paste error, will fix it